docs/INSTALLATION.md

```diff
@@ -24,7 +24,7 @@
 
 | Plugin version | Cordova CLI | Cordova Android | Cordova iOS | CocoaPods |
 | -------------- | ----------- | --------------- | ----------- | --------- |
-| 1.0.0          | 10.0.0      | 8.0.0           | 5.1.1       | 1.8.0     |
+| 1.0.0          | 10.0.0      | 8.0.0           | 6.0.0       | 1.8.0     |
```

Comment:
this should be 2.0.0 or add a new line for 2.0.0 requirements

Reply:
Fixed